eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -174,13 +174,18 @@ stages:
                             PackageInfoLocations:
                               - $(Pipeline.Workspace)/${{parameters.ArtifactName}}/PackageInfo/${{artifact.name}}.json
                             WorkingDirectory: $(System.DefaultWorkingDirectory)
-                            TargetDocRepoOwner: ${{parameters.TargetDocRepoOwner}}
+                            TargetDocRepoOwner: Azure
```

Comment:
How did this work before given TargetDocRepoOwner parameter didn't exist? Also as Heath mentioned since you have defaulted this to Azure you don't need to explicitly pass it.

Reply:
Yes, I am trying to understand this piece as well. Distracted by some other work.

Reply:
This is inherited from `archetype-sdk-client.yml`. release yml is the only place which we declare the docs repo, so I decided to hard coded here.